cypress/utils/dashboards/workspace-plugin/constants.js

```diff
@@ -4,3 +4,8 @@
  */
 
 export const WORKSPACE_API_PREFIX = '/api/workspaces';
+
+export const NONE_DASHBOARDS_ADMIN_USER = {
```

Comment:
What about create internal user with command `createInternalUser` so the workspace tests don't rely on specific user and pass.

Reply:
I've tried this way before. But not only do we need to call createInternalUser, we also need to create a role and map the internal user to it too. It will be more complicated.